static/js/components/EmploymentTab.js

```diff
@@ -235,10 +235,7 @@ class EmploymentTab extends ProfileTab {
   }
 
   render () {
-    const {
-      ui: { workHistoryEdit },
-      ui: { workDialogVisibility },
-    } = this.props;
+    const { ui: { workHistoryEdit }, ui: { workDialogVisibility } } = this.props;
```

Comment:
You should be able to merge this further: `{ ui: { workHistoryEdit, workDialogVisibility } }`